java/code/src/com/suse/manager/webui/services/SaltServerActionService.java

```diff
@@ -337,21 +337,24 @@ else if (ActionFactory.TYPE_APPLY_STATES.equals(actionType)) {
         }
         else if (ActionFactory.TYPE_IMAGE_INSPECT.equals(actionType)) {
             ImageInspectAction iia = (ImageInspectAction) actionIn;
-            ImageInspectActionDetails details = iia.getDetails();
+            Optional<ImageInspectActionDetails> details = Optional.ofNullable(iia.getDetails());
+            if (details.isEmpty()) {
+                return new HashMap<>();
```

Comment:
Any reason why you are creating a new instance instead of using `Collections.emptyMap()` like you do in the  `orElseGet()`  line below? This inconsistency may lead to an issue if the caller has to change the collection since `Collections.emptyMap()` is immutable.